src/main/java/app/IDEAppBuilder.java

```diff
@@ -126,10 +125,6 @@ public JTabbedPane makeEditorPanel() {
         return leftEditorTabbedPane;
     }
 
-    /*public void openFile(File file) {
-        leftEditorTabbedPane.addTab(file);
-    }*/
-
     public void openFile(File file) {
```

Comment:
For the openFile method maybe add an else statement in case the file is invalid or can't be opened? I currently have it in my branch will show you the changes when i make a pull request